Add bin/magento reference topics (#4693)
magento/devdocs: _includes/config/cli-reference-template.md

{% assign app = file.application %}

{% assign commands = file.commands %}
{% if app %}
# {{ app.name }} (version {{ app.version }})
{% else %}
# Magento CLI
{% endif %}

This reference contains {{ commands | size }} commands available through the `bin/magento` command-line tool.

> The initial list is auto generated using `bin/magento list` command at the {{ site.data.var.ce }} edition.

Reviewer: ```suggestion
The initial list is autogenerated using the `bin/magento list` command on the {{ site.data.var.ce }} edition.
```
